.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,29 +1,12 @@
 # Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
 blank_issues_enabled: false  # default: true
 contact_links:
-- name: 🔐 Security bug report 🔥
```

Comment:
I'd probably keep this because it emphasizes that people shouldn't be posting vulnerabilities in public

Reply:
Please open https://github.com/aio-libs/aiohttp/issues/new/choose
The third and fourth items point to the same link.

Reply:
Yes, unfortunately the first one is less visible so I tend to emphasize what to do with a bigger box.